stock_available_stock_mts_mto_rule/models/stock_rule.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class StockRule(models.Model):
+    _inherit = "stock.rule"
+
+    @api.model
+    def _get_available_quantity_mts_mto_rule(self, product):
+        product = product.with_context(location=False)
```

Comment:
Sorry this is not correct. We should not remove the location from the context.
Otherwise all of the stock locations are used, to compute the qty. 
```suggestion
```